x-pack/plugins/cases/common/api/connectors/index.ts

```diff
@@ -21,7 +21,9 @@ export * from './resilient';
 export * from './mappings';
 export * from './swimlane';
 
-export type ActionConnector = ActionResult;
+// This marks the config field as partial because preconfigured connectors will not have a config section
+// see: https://github.com/elastic/kibana/issues/119696 for more details
+export type ActionConnector = Omit<ActionResult, 'config'> & Partial<Pick<ActionResult, 'config'>>;
```

Comment:
@cnasikas do you know of a helper/utility type that already does this? aka marks a single property within a type as partial?

Reply:
As far I remember, no.